distribution/lib/Standard/Base/0.0.0-dev/src/Data/Json.enso

```diff
@@ -114,12 +114,13 @@ type JS_Object
        Values will be recursively converted to JSON serializable as needed.
     from_pairs : Vector -> JS_Object
     from_pairs pairs =
-        js_object = pairs.fold new_object current->pair->
+        js_object_fn = pairs.fold new_object current->pair->
             case pair.first of
                 text : Text ->
                     js_value = pair.second.to_js_object
                     set_value current text js_value
                 _ -> Error.throw (Illegal_Argument.Error "JS_Object.from_pairs: key must be a Text value")
+        js_object = js_object_fn
```

Comment:
I'd like to understand what this changes. To my intuition this should be the same before and after, no?

Is `js_object_fn` a zero-argument function block? From my understanding it should be a simple value.